src/dstack/_internal/core/backends/oci/region.py

```diff
@@ -0,0 +1,38 @@
+from functools import cached_property
+
+import oci
+from typing_extensions import Any, List, Mapping
+
+from dstack._internal.core.backends.oci.auth import get_client_config
+from dstack._internal.core.models.backends.oci import AnyOCICreds
+
+
+class OCIRegion:
```

Comment:
I suggest calling this class something other than just OCIRegion (e.g. OCIRegionClient) to avoid confusing it with variables for region names.